src/test/resources/app.properties

```diff
@@ -0,0 +1,2 @@
+chromeDriver = webdriver.chrome.driver
+chromeDriverPath = /Users/avoloshchuk/IdeaProjects/com.sharelane/src/test/resources/chromedriver79
```

Comment:
%)
what do you think will happened when I get your project and try to run it?

Reply:
Path was changed to
new File("src/test/resources/app.properties").getAbsolutePath();